client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -83,6 +83,11 @@ button {
 		height: 8px;
 		background-color: transparent;
 
+		&.is-fixed {
```

Comment:
I think the fixed progress bar is only shown on the desktop

Reply:
I'll check